release/tune_tests/cloud_tests/workloads/run_cloud_test.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         trial_id: str,
         status: str,
         config: Dict[str, Any],
-        local_dir: str,
+        _local_dir: str,
```

Comment:
I wonder if it's possible to write basic unit tests for the release tests to catch these kind of errors earlier?

Reply:
It looks like some AIR/ML release tests have smoke tests, but not all of our release tests have a smoke test option.